packages/block-library/src/quote/edit.js

```diff
@@ -24,12 +24,14 @@ export default function QuoteEdit( {
 	onReplace,
 	className,
 	insertBlocksAfter,
+	mergedStyle,
 } ) {
 	const { align, value, citation } = attributes;
 	const blockProps = useBlockProps( {
 		className: classnames( className, {
 			[ `has-text-align-${ align }` ]: align,
 		} ),
+		style: mergedStyle,
```

Comment:
I'm just wondering if that doesn't break anything on the web. I believe the `mergedStyle` is `undefined` for the web but it would be good to confirm that:)

Reply:
Thanks for double for checking! I tested this change on the web editor and it works without any issues, we already have this use case in the [Heading](https://github.com/WordPress/gutenberg/blob/master/packages/block-library/src/heading/edit.js#L37) block.